docs/reference/catalog.md

```diff
@@ -1,8 +1,16 @@
-# Catalog class
+# Catalog
+
+## Catalog object
 
 The Catalog class enables access to the UP42 catalog search. You can search
 for satellite image scenes (for different sensors and criteria like cloud cover),
 plot the scene coverage and download and plot the scene quicklooks.
 
 
-::: up42.catalog.Catalog
\ No newline at end of file
+::: up42.catalog.Catalog
+    rendering:
+        show_root_toc_entry: False
+    selection:
+        inherited_members: True
+        members: ["construct_parameters", "search", "download_quicklooks", "plot_quicklooks", "map_quicklooks"]
```

Comment:
Less then ideal, but currently no way to exclude specific members, only via not including them. Feature request in mkdocstrings: https://github.com/pawamoy/mkdocstrings/issues/157